tests/test_condor_unit.py

```diff
@@ -115,25 +115,29 @@ class TestCondorUnit:
 
     # lib/condor.py routines...
 
+    @pytest.mark.unit
     def test_get_schedd_1(self):
         """make sure we get our test schedd back with test_vargs"""
         schedd = condor.get_schedd(TestUnit.test_vargs)
         print("got schedd: {0}".format(schedd))
         print("schedd name: {0}".format(schedd["Name"]))
         assert schedd["Name"] == TestUnit.test_schedd
 
+    @pytest.mark.unit
```

Comment:
In my mind, this is a true unit test, in that it doesn't require our collector pool to actually work.